rareapi/views/Profiles.py

```diff
@@ -6,12 +6,25 @@
 from rest_framework import status
 from django.core.exceptions import ValidationError
 from rest_framework import serializers
+from django.http.response import HttpResponseServerError
 
 class ProfileViewSet(ViewSet):
     def list(self, request):
         users = RareUsers.objects.all()
         serializer = BasicProfileSerializer(users, many=True, context={'request':request})
         return Response(serializer.data)
+    
+    def retrieve(self, request, pk=None):
+        """Handle GET request for single post
```

Comment:
This docstring should probably talk about single rare_user not single post

Reply:
updated